src/routes/api/oba/routes/+server.js

```diff
@@ -0,0 +1,39 @@
+import oba from '$lib/obaSdk.js';
+
+// TODO: change this api after deployment of get all routes api
+export async function GET() {
+	try {
+		const agenciesResponse = await oba.agenciesWithCoverage.list();
```

Comment:
it doesn't look like you're caching this data on the server side yet, right? This should be run once per app launch—and then cached in memory, instead of reloaded on every request.